duckduckgo/iOS
# network leaderboard tweaks

Task/Issue URL: https://app.asana.com/0/414235014887631/494552239039546
Tech Design URL:
CC:

**Description**:

Minor optimisation to the network leaderboard code and vertical centering of the top offenders.

**Steps to test this PR**:
1. Fresh install (or reset the network tracker) 
1. Visit DuckDuckGo.com
1. Open dashboard, observe vertical centering and new 'no stats' icon with still gathering message
1. Visit thehill.com
1. Visit cnn.com
1. Open dashboard, observe vertical centering of the top offenders title and pills

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
